README.md

```diff
@@ -14,7 +14,7 @@ compile 'com.stepstone.apprating:app-rating:2.0.0'
 ```
 
 ## Supported features
-  - colors for dialog's background, title, description
+  - colors for dialog's background, title, description, hint
```

Comment:
You could also add that changing the hint text is also possible :)

Reply:
right, thanks